services/terrain/src/terrain/clients/data_info.clj

```diff
@@ -170,12 +170,10 @@
     (raw/move-single user path-uuid dest)))
 
 (defn move
-  "Uses the data-info single and bulk mover endpoints to move an item or many items into a new directory."
+  "Uses the data-info bulk mover endpoint to move items into a new directory."
   [{:keys [user]} {:keys [sources dest]}]
   (validators/not-superuser user)
-  (if (= 1 (count sources))
-    (move-single user (first sources) dest)
-    (raw/move-multi user sources dest)))
+  (raw/move-multi user sources dest))
```

Comment:
Yes, the different response for 1 file/folder caused the UI to throw an exception when attempting to parse the new response, since it was always expecting the `sources` array in the response from the [POST /filesystem/move endpoint](https://github.com/cyverse/DE/blob/gh-pages-beta/api/endpoints/filesystem/move.md#moving-files-andor-directories).